src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -55,20 +56,31 @@ class WorkspaceSettingsPage extends React.Component {
         if (this.props.policy.isPolicyUpdating) {
             return;
         }
-        const name = values.name.trim();
         const outputCurrency = values.currency;
-        Policy.updateGeneralSettings(this.props.policy.id, name, outputCurrency);
+        Policy.updateGeneralSettings(this.props.policy.id, values.name, outputCurrency);
         Keyboard.dismiss();
     }
 
     validate(values) {
         const errors = {};
-        if (!values.name || !values.name.trim().length) {
+        const name = values.name.trim();
+
+        if (this.hasHtml(name)) {
+            errors.name = 'HTML tags are not allowed in workspace names.';
```

Comment:
Does the API enforce this too? 🤔

Reply:
So currently the WAF input rules strip the html from the variable, as its under `FREETEXT`

Reply:
To allow html through, it looks like we need to downgrade the type to `ANYTHING`. 

I'd say, since html doesn't make it to php, lets just filter it out here. If it does get through somehow, it will be stripped, and whats left is saved as a workspace name.

Reply:
Yeah that's fine